draft-wang-ppm-differential-privacy.md

```diff
@@ -255,12 +255,58 @@ with the correct noise type.
 > TODO: Chris P: we will mention Prio3SumVec because that's what we use to describe aggregator DP with amplification
 
 
-# Differential Privacy Mechanism {#dp mechanism}
+# Differential Privacy Mechanism {#dp-mechanism}
 
 This section should define a framework to apply different dp noise type to a
 client measurement and/or an aggregation.
 
-> TODO: Junye to spell out the interface of dp mechanisms, "Noise Mechanisms to Accomplish DP"
+Clients and Aggregators can implement various noise mechanisms to accomplish
```

Comment:
Noise mechanisms and DP mechanisms are mixed, we should stick to one of them, I’d vote for DP mechanisms

Reply:
Changed all to DP mechanisms